mscore/file.cpp

```diff
@@ -2328,7 +2328,7 @@ Score::FileError readScore(MasterScore* score, QString name, bool ignoreVersionE
                   { "gpx",  &importGTP                },
                   { "ptb",  &importGTP                },
 #ifdef AVSOMR
-                  { "avs",  &importAVS                },
+                  { "msmr", &importAVS                },
```

Comment:
Not `&importMSMR`?

Reply:
I thought to rename or not to rename. On the one hand it is mainly the import of audiveris (avs), like importGTP or importOve, therefore named importAVS. On the other hand it will probably be clear if be rename it to importMsmr.
(we ourselves create a .msmr file (we design a format), so we can give it a more understandable extension for users)

I tend to rename.   
   
By the way, I would like to have that the import of third-party formats and reading of native (.msсz, .mscx) do not differ in the structure of the code, but differ only in the implementation of the reader.  And different readers implemented one interface. Also with writers.